database/seeder.js

```diff
@@ -0,0 +1,99 @@
+const mongoose = require('mongoose');
+const progress = require('cli-progress');
+const db = require('./index.js');
+
+
+/******* PROGRESS BAR *******/
+const bar = new progress.Bar({}, progress.Presets.shades_classic);
+let completed = 0;
+bar.start(122, 0);
+
+
+/********  SAMPLE PERSONNEL DATA ********/
+const actors = ["Elvis Presley","John Travolta","John Belushi","George Clooney","Natalie Portman","Scarlett Johannsson","Marlon Brando","Margot Robbie","Will Smith","Chris Evans","Robert Downey Jr","Jon Bon Jovi","Miley Cyrus","Keanu Reeves","Lawrence Fishburne","Samuel L. Jackson","Brad Pitt","Angelina Jolie","Emma Watson","Alan Rickman","Benedict Cumberbatch","Benedict Wong"];
```

Comment:
Seems odd that we're bouncing back and forth btwn `'` vs `"`. Are you using a linter?